engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/versions/views/content-view-version-yum.html

```diff
@@ -4,13 +4,30 @@
       <tr bst-table-head>
         <th bst-table-column translate>Name</th>
         <th bst-table-column translate>Product</th>
+        <th bst-table-column translate>Content</th>
       </tr>
     </thead>
 
     <tbody>
       <tr bst-table-row ng-repeat="repository in table.rows">
         <td bst-table-cell>{{ repository.name }}</td>
         <td bst-table-cell>{{ repository.product.name }}</td>
+        <td bst-table-cell>
+          <div>
+            <a ui-sref="products.repository.manage-content.packages({productId: repository.product.id, repositoryId: repository.id})"
+               ng-show="repository.content_type == 'yum'"
```

Comment:
Seems like this `ng-show` could just go on the table cell itself or a containing div?  Also, what happens when the content type isn't yum?